### PR TITLE
Was added code from the project completed during the Admissions Trial

### PR DESCRIPTION
**Pull request by SergSkol**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
```
Comment (SergSkol) [inline index.html:4]: I would recommend you use 2 spaces indent instead of 4 spaces/tab.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <main id=""main>
```
Comment (SergSkol) [inline index.html:9]: It seems to be a typo, the name should be inside quotes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <main id=""main>
+        <h1 id="title">Dr. Norman Borlaug</h1>
```
Comment (SergSkol) [inline index.html:10]: How about to use class in addition to id?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <title>Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <main id=""main>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+            <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
```
Comment (SergSkol) [inline index.html:14]: This string seems too long, you can split it to 2-3 lines for better readability.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        </div>
+        <div id="tribute-info">
+
+          <section id="Life-info">
+            <h1> Here's a time line of Dr. Borlaug's life:</h1>
+            <ul>
+              <li><strong>1914</strong>- Born in Cresco, Iowa </li>
+              <li><strong>1933</strong>- - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration" </li>
+            </ul>
+
```
Comment (SergSkol) [inline index.html:24]: Such empty lines are not required, could you please remove it?

File: style.css [hunk 6]
```diff
@@ -0,0 +1,7 @@
+#image {
+  max-width: 460px;
+  height: auto;
+}  
+img {
```
Comment (SergSkol) [inline style.css:5]: Please add an empty line between tags.

**Review by SergSkol — COMMENTED** [review]
Code looks fine, but some changes are required to make it better, please find my comments below.

File: index.html (hunk 1 above)
Comment (Olamarx) [reply index.html:4]: [OTIONAL] It would be better couched as 'I would recommend you use...'

**Review by Olamarx — APPROVED** [review]
Hi @SergSkol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of inline comments ✔️
- Good choice of words 👍🏼
- Grammatically correct sentences 😉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**